### PR TITLE
Transform metric scope into tags

### PR DESCRIPTION
**Pull request by petercable**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,5 +27,5 @@ kafka.metrics.reporters=com.appoptics.integrations.kafka.broker.KafkaAppopticsRe
 appoptics.token=[api-token]
 appoptics.agent.identifier=[hostname]
 appoptics.tags=[tag1name=tagvalue,tag2name=tag2value]
-# Tag name/value restrictions located: https://docs.appoptics.com/api/#measurement-restrictions
+# Tag NAME/value restrictions located: https://docs.appoptics.com/api/#measurement-restrictions
```
Comment (mheffner) [inline README.md]: Was this intended?

File: src/main/java/com/appoptics/integrations/kafka/broker/DeltaMetricSupplier.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        this.predicate = predicate;
+    }
+
+    public Map<NameAndTags, Metric> getMetrics() {
+        final Map<NameAndTags, Metric> map = new HashMap<>();
+        for (Map.Entry<String, SortedMap<MetricName, Metric>> entry : registry.groupedMetrics(predicate).entrySet()) {
+            for (Map.Entry<MetricName, Metric> metricEntry : entry.getValue().entrySet()) {
+                NameAndTags nameAndTags = new NameAndTags(metricEntry.getKey());
+                map.put(nameAndTags, metricEntry.getValue());
+            }
```
Comment (mheffner) [inline src/main/java/com/appoptics/integrations/kafka/broker/DeltaMetricSupplier.java]: Maybe use streams here?

**Review by mheffner — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (petercable) [reply README.md]: refactor overflow, fixed.

File: src/main/java/com/appoptics/integrations/kafka/broker/DeltaMetricSupplier.java (hunk 2 above)
Comment (petercable) [reply src/main/java/com/appoptics/integrations/kafka/broker/DeltaMetricSupplier.java]: agreed!

**Review by lucky — APPROVED** [review]

**Review by lucky — APPROVED** [review]

**Pull request merged**